ci/build_python.sh

```diff
@@ -42,16 +41,6 @@ rapids-conda-retry mambabuild \
   --channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" \
   conda/recipes/cugraph
```

Comment:
Why does this repo build cugraph? Shouldn’t it only build GNN related packages?

Reply:
You're right, this repo should only be building `cugraph-dgl`, `cugraph-pyg`, `libwholegraph`, and `pylibwholegraph`.

Since you commented on it, I just pushed that change to this specific script.

If you look around this repo you'll see many other unnecessary things. This repo was created by copying big chunks of the `cugraph` and `wholegraph` repos, and there's lots of such cleanup needed. #53 is trying to do that all at once and I think it's dragging as a result... this PR is just trying to take a small, incremental step forward to reduce the size of that effort.

Reply:
Oh. I realize now that this PR isn't intending to do builds at all, it is really "minimal" CI (just style checks). In that case, I'll approve this as-is.